mobile/android/Interface.cpp

```diff
@@ -111,6 +111,7 @@ Java_org_iota_mobile_Interface_iota_1sign_1address_1gen_1trytes(
   char const* seed = env->GetStringUTFChars(jseed, 0);
 
   if ((address = iota_sign_address_gen_trytes(seed, index, security)) == NULL) {
+    memset_safe((void*)seed, 81, 0, 81);
```

Comment:
you should call `ReleaseStringUTFChars` when `seed` is not used any more otherwise, it would result in a memory leak.